build/helm/keylime/charts/keylime-agent/templates/daemonset.yaml

```diff
@@ -8,6 +8,9 @@ spec:
   selector:
     matchLabels:
       {{- include "agent.selectorLabels" . | nindent 6 }}
+  {{- if .Values.serviceAccount.create }}
+  serviceAccountName: {{ include "agent.serviceAccountName" . }}
+  {{- end }}
```

Comment:
`serviceAccountName` should be part of the spec, and I can see it already below here. Or is there something new going on which I'm missing?